Node segment `.Mismatch` and Node codenames

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

Some node projects that I use recently changed their `.nvmrc` files to use the LTS codenames instead of an actual version number. e.g. previously, the contents of `.nvmrc` was `v20.10.0`, whereas now it is `lts/iron`.

At the moment, NVM resolves `lts/iron` to `v20.10.0`, but the oh-my-posh `node` segment doesn't in its mismatch logic.

e.g. here is my segment:

![CleanShot 2023-12-19 at 12 32 59@2x](https://github.com/JanDeDobbeleer/oh-my-posh/assets/4434330/d614c3b9-8da9-448e-9c3b-6d662e1119cf)

Is it possible for the segment to resolve the LTS codename in the `.Mismatch` property's logic? 

Thanks for the consideration!

### Theme

Here is my segment's config:

```
        {
          "type": "node",
          "style": "powerline",
          "powerline_symbol": "",
          "foreground": "#ffffff",
          "background": "#6CA35E",
          "template": " {{ if .Mismatch }}{{ .Full }} 󰦎 {{ .Expected }} {{ end }}"
        },
```

### What OS are you seeing the problem on?

macOS

### Which shell are you using?

zsh

### Log output

```Shell
❯ oh-my-posh debug --plain

Version: 19.2.1

Shell: zsh (5.9)

Prompt:

─────────────────────────────────────────────────────────────────────────────────────────────────────────
 󱓞   repo  20.10.0 󰦎 lts/iron   main ≡ 
❯

Segments:

ConsoleTitle(false)                       -   0 ms
Command(true)                             -   9 ms
Exit(false)                               -   1 ms
Text(true)                                -   0 ms
Path(true)                                -   1 ms
Node(true)                                -  20 ms
Git(true)                                 -  51 ms
Text(true)                                -   0 ms

Run duration: 63.255875ms

Cache path: /Users/lucascosti/.cache/oh-my-posh

Config path: /Users/lucascosti/repos/mac-config/zsh/zshscripts/themes/oh-my-posh-lucas.omp.json

Logs:

[DEBUG] 12:37:48.693 shell.go:Getenv:399 → iTerm.app
[TRACE] 12:37:48.693 shell.go:Getenv(TERM_PROGRAM) - 22.542µs
[DEBUG] 12:37:48.693 shell.go:Getenv:399 → NO DATA
[TRACE] 12:37:48.693 shell.go:Getenv(OMP_CACHE_DIR) - 2.084µs
[DEBUG] 12:37:48.693 shell.go:Getenv:399 → NO DATA
[TRACE] 12:37:48.693 shell.go:Getenv(XDG_CACHE_HOME) - 1.667µs
[TRACE] 12:37:48.693 shell_unix.go:CachePath() - 31.125µs
[DEBUG] 12:37:48.693 shell.go:Getenv:399 → /Users/lucascosti/repos/mac-config/zsh/zshscripts/themes/oh-my-posh-lucas.omp.json
[TRACE] 12:37:48.693 shell.go:Getenv(POSH_THEME) - 5µs
[DEBUG] 12:37:48.693 shell_unix.go:Platform:77 → darwin
[TRACE] 12:37:48.693 shell.go:resolveConfigPath() - 11.666µs
[TRACE] 12:37:48.693 shell.go:Init() - 199.416µs
[TRACE] 12:37:48.693 shell.go:Flags() - 83ns
[TRACE] 12:37:48.693 config.go:loadConfig() - 371.25µs
[TRACE] 12:37:48.693 shell.go:Flags() - 41ns
[DEBUG] 12:37:48.693 shell.go:Getenv:399 → NO DATA
[TRACE] 12:37:48.693 shell.go:Getenv(OMP_CACHE_DISABLED) - 1.875µs
[DEBUG] 12:37:48.693 shell.go:Shell:662 → no shell name provided in flags, trying to detect it
[DEBUG] 12:37:48.693 shell.go:Shell:670 → process name: zsh
[TRACE] 12:37:48.693 shell.go:Shell() - 176.916µs
[DEBUG] 12:37:48.693 shell.go:Getenv:399 → 5.9
[TRACE] 12:37:48.693 shell.go:Getenv(POSH_SHELL_VERSION) - 1.917µs
[DEBUG] 12:37:48.693 debug.go:PrintDebug:22 → Segment: Title
[DEBUG] 12:37:48.693 text.go:Render:70 → Rendering template:
[DEBUG] 12:37:48.693 shell.go:Getenv:399 → 1
[TRACE] 12:37:48.693 shell.go:Getenv(POSH_CURSOR_LINE) - 2.917µs
[DEBUG] 12:37:48.693 shell.go:Getenv:399 → 1
[TRACE] 12:37:48.693 shell.go:Getenv(POSH_CURSOR_COLUMN) - 1.583µs
[TRACE] 12:37:48.693 shell.go:Flags() - 42ns
[TRACE] 12:37:48.693 shell.go:Flags() - 0s
[TRACE] 12:37:48.693 shell.go:Shell() - 42ns
[TRACE] 12:37:48.693 shell.go:Shell() - 42ns
[TRACE] 12:37:48.693 shell.go:Flags() - 42ns
[TRACE] 12:37:48.693 shell.go:GOOS() - 41ns
[DEBUG] 12:37:48.693 shell.go:Pwd:429 → /Users/lucascosti/repos/repo
[TRACE] 12:37:48.693 shell.go:Pwd() - 14.416µs
[TRACE] 12:37:48.693 shell.go:GOOS() - 41ns
[DEBUG] 12:37:48.693 segment.go:SetEnabled:512 → Segment: Command
[DEBUG] 12:37:48.693 properties.go:GetString:28 → zsh
[DEBUG] 12:37:48.693 shell.go:CommandPath:615 → /bin/zsh
[TRACE] 12:37:48.693 shell.go:CommandPath(zsh) - 73.75µs
[TRACE] 12:37:48.693 shell.go:HasCommand(zsh) - 76µs
[DEBUG] 12:37:48.693 properties.go:GetString:28 ↓
    printf '%*s
    ' "${COLUMNS:-$(tput cols)}" '' | tr ' ' ─
[DEBUG] 12:37:48.703 shell.go:RunCommand:594 → ─────────────────────────────────────────────────────────────────────────────────────────────────────────
[TRACE] 12:37:48.703 shell.go:RunCommand(zsh -c printf '%*s
' "${COLUMNS:-$(tput cols)}" '' | tr ' ' ─) - 9.276375ms
[TRACE] 12:37:48.703 shell.go:RunShellCommand() - 9.298333ms
[TRACE] 12:37:48.703 shell_unix.go:Root() - 542ns
[TRACE] 12:37:48.703 shell.go:Shell() - 916ns
[TRACE] 12:37:48.703 shell.go:StatusCodes() - 42ns
[DEBUG] 12:37:48.703 shell_unix.go:IsWsl:35 → false
[TRACE] 12:37:48.703 shell_unix.go:IsWsl() - 7.042µs
[DEBUG] 12:37:48.703 shell.go:TemplateCache:843 → environment: 
<<<redacted>>>
[TRACE] 12:37:48.703 shell.go:Pwd() - 125ns
[TRACE] 12:37:48.703 shell.go:GOOS() - 42ns
[TRACE] 12:37:48.703 shell.go:GOOS() - 41ns
[TRACE] 12:37:48.703 shell.go:GOOS() - 42ns
[TRACE] 12:37:48.703 shell.go:GOOS() - 83ns
[TRACE] 12:37:48.703 shell.go:GOOS() - 42ns
[TRACE] 12:37:48.703 shell.go:GOOS() - 41ns
[TRACE] 12:37:48.703 shell.go:GOOS() - 41ns
[TRACE] 12:37:48.703 shell.go:GOOS() - 0s
[TRACE] 12:37:48.703 shell.go:GOOS() - 41ns
[TRACE] 12:37:48.703 shell.go:GOOS() - 42ns
[TRACE] 12:37:48.703 shell.go:GOOS() - 41ns
[DEBUG] 12:37:48.703 shell.go:User:564 → lucascosti
[TRACE] 12:37:48.703 shell.go:User() - 2.375µs
[DEBUG] 12:37:48.703 shell.go:Host:576 → lccanvalappyv1.FrostyLAN
[TRACE] 12:37:48.703 shell.go:Host() - 13µs
[TRACE] 12:37:48.703 shell.go:GOOS() - 0s
[DEBUG] 12:37:48.703 shell.go:Getenv:399 → 1
[TRACE] 12:37:48.703 shell.go:Getenv(SHLVL) - 2.167µs
[TRACE] 12:37:48.703 shell.go:TemplateCache() - 148.75µs
[DEBUG] 12:37:48.703 text.go:Render:70 → Rendering template: {{ .Output }}
[TRACE] 12:37:48.703 shell.go:TemplateCache() - 125ns
[TRACE] 12:37:48.703 shell.go:Shell() - 42ns
[DEBUG] 12:37:48.703 text.go:Render:70 → Rendering template: plain
[TRACE] 12:37:48.703 shell.go:Shell() - 42ns
[TRACE] 12:37:48.703 shell.go:Shell() - 42ns
[TRACE] 12:37:48.703 shell.go:Shell() - 42ns
[TRACE] 12:37:48.703 shell.go:Shell() - 42ns
[TRACE] 12:37:48.703 shell.go:Flags() - 0s
[TRACE] 12:37:48.703 shell.go:Pwd() - 83ns
[TRACE] 12:37:48.703 shell.go:GOOS() - 0s
[DEBUG] 12:37:48.703 segment.go:SetEnabled:512 → Segment: Git
[DEBUG] 12:37:48.703 shell_unix.go:IsWsl:35 → false
[TRACE] 12:37:48.703 shell_unix.go:IsWsl() - 2.083µs
[TRACE] 12:37:48.703 shell_unix.go:IsWsl2() - 4µs
[TRACE] 12:37:48.703 shell.go:Flags() - 166ns
[TRACE] 12:37:48.703 shell.go:GOOS() - 42ns
[TRACE] 12:37:48.703 shell.go:Pwd() - 84ns
[TRACE] 12:37:48.703 shell.go:GOOS() - 41ns
[DEBUG] 12:37:48.703 segment.go:SetEnabled:512 → Segment: Text
[TRACE] 12:37:48.703 shell.go:Flags() - 333ns
[TRACE] 12:37:48.703 shell.go:TemplateCache() - 83ns
[TRACE] 12:37:48.703 shell.go:Flags() - 41ns
[DEBUG] 12:37:48.703 shell.go:CommandPath:615 → /Users/lucascosti/.nix-profile/bin/git
[TRACE] 12:37:48.703 shell.go:CommandPath(git) - 41.416µs
[TRACE] 12:37:48.703 shell.go:HasCommand(git) - 43.666µs
[DEBUG] 12:37:48.703 properties.go:GetString:28 → {{ .Code }}
[TRACE] 12:37:48.703 shell.go:Pwd() - 42ns
[TRACE] 12:37:48.703 shell.go:Pwd() - 0s
[TRACE] 12:37:48.703 shell.go:Pwd() - 125ns
[TRACE] 12:37:48.703 shell.go:GOOS() - 42ns
[TRACE] 12:37:48.703 shell.go:GOOS() - 42ns
[TRACE] 12:37:48.703 shell.go:StatusCodes() - 84ns
[DEBUG] 12:37:48.703 text.go:Render:70 → Rendering template: {{ .Code }}
[TRACE] 12:37:48.703 shell.go:HasParentFilePath(.git) - 9.875µs
[DEBUG] 12:37:48.703 segment.go:SetEnabled:512 → Segment: Path
[TRACE] 12:37:48.703 shell.go:Pwd() - 42ns
[DEBUG] 12:37:48.703 properties.go:GetStringArray:52 → exclude_folders: []
[TRACE] 12:37:48.703 shell.go:Shell() - 42ns
[TRACE] 12:37:48.703 shell.go:GOOS() - 42ns
[TRACE] 12:37:48.703 shell.go:GOOS() - 84ns
[TRACE] 12:37:48.703 shell.go:Shell() - 41ns
[DEBUG] 12:37:48.703 properties.go:GetBool:22 → fetch_user: false
[TRACE] 12:37:48.703 shell.go:GOOS() - 42ns
[TRACE] 12:37:48.703 shell.go:GOOS() - 42ns
[TRACE] 12:37:48.703 shell.go:GOOS() - 42ns
[TRACE] 12:37:48.703 shell.go:GOOS() - 42ns
[TRACE] 12:37:48.703 shell.go:GOOS() - 42ns
[TRACE] 12:37:48.703 shell.go:Flags() - 125ns
[TRACE] 12:37:48.703 shell.go:GOOS() - 41ns
[TRACE] 12:37:48.703 properties.go:GetBool:2[TRACE] 12:37:48.703 shell.go:GOOS() - 42ns
[DEBUG] 12:37:48.703 properties.go:GetString:28 → 
[TRACE] 12:37:48.703 shell.go:GOOS() - 41ns
[DEBUG] 12:37:48.703 properties.go:GetString:28 → 
[TRACE] 12:37:48.703 shell.go:GOOS() - 42ns
[DEBUG] 12:37:48.703 shell.go:Getenv:399 → NO DATA
[TRACE] 12:37:48.703 shell.go:Getenv(POSH_GIT_STATUS) - 2.416µs
[DEBUG] 12:37:48.703 properties.go:GetString:28 → ~
[DEBUG] 12:37:48.703 properties.go:GetBool:22 → fetch_status: true
[DEBUG] 12:37:48.703 properties.go:GetStringArray:52 → ignore_status: []
[TRACE] 12:37:48.703 shell.go:GOOS() - 42ns
[TRACE] 12:37:48.703 shell.go:Pwd() - 83ns
[TRACE] 12:37:48.703 shell.go:GOOS() - 42ns
[DEBUG] 12:37:48.703 properties.go:GetKeyValueMap:46 → status_formats: map[]
[DEBUG] 12:37:48.703 properties.go:GetKeyValueMap:46 → mapped_locations: map[~/repos/:/]
[DEBUG] 12:37:48.703 properties.go:GetKeyValueMap:46 → untracked_modes: map[]
[DEBUG] 12:37:48.703 segment.go:SetEnabled:512 → Segment: Node
[DEBUG] 12:37:48.703 text.go:Render:70 → Rendering template: ~/repos/
[TRACE] 12:37:48.703 shell.go:GOOS() - 41ns
[DEBUG] 12:37:48.703 properties.go:GetKeyValueMap:46 → ignore_submodules: map[]
[TRACE] 12:37:48.703 shell.go:PathSeparator() - 84ns
[TRACE] 12:37:48.703 shell.go:GOOS() - 42ns
[TRACE] 12:37:48.703 shell.go:GOOS() - 42ns
[DEBUG] 12:37:48.703 properties.go:GetStringArray:52 → extensions: [*.js *.ts package.json .nvmrc pnpm-workspace.yaml .pnpmfile.cjs .npmrc .vue]
[DEBUG] 12:37:48.703 properties.go:GetStringArray:52 → folders: []
[DEBUG] 12:37:48.703 properties.go:GetBool:22 → home_enabled: false
[TRACE] 12:37:48.703 shell.go:Pwd() - 41ns
[DEBUG] 12:37:48.703 properties.go:GetString:28 → files
[DEBUG] 12:37:48.703 properties.go:GetBool:22 → fetch_package_manager: false
[TRACE] 12:37:48.703 shell.go:Pwd() - 84ns
[TRACE] 12:37:48.704 shell.go:GOOS() - 125ns
[TRACE] 12:37:48.704 shell.go:GOOS() - 84ns
[TRACE] 12:37:48.704 shell.go:PathSeparator() - 41ns
[TRACE] 12:37:48.704 shell.go:PathSeparator() - 41ns
[TRACE] 12:37:48.704 shell.go:GOOS() - 42ns
[TRACE] 12:37:48.704 shell.go:GOOS() - 42ns
[TRACE] 12:37:48.704 shell.go:PathSeparator() - 42ns
[TRACE] 12:37:48.704 shell.go:GOOS() - 42ns
[TRACE] 12:37:48.704 shell.go:GOOS() - 42ns
[TRACE] 12:37:48.704 shell.go:PathSeparator() - 83ns
[TRACE] 12:37:48.704 shell.go:GOOS() - 41ns
[TRACE] 12:37:48.704 shell.go:GOOS() - 42ns
[TRACE] 12:37:48.704 shell.go:PathSeparator() - 42ns
[TRACE] 12:37:48.704 shell.go:PathSeparator() - 0s
[DEBUG] 12:37:48.704 properties.go:GetString:28 → agnoster_short
[TRACE] 12:37:48.704 shell.go:PathSeparator() - 42ns
[DEBUG] 12:37:48.704 shell.go:HasFiles:460 → true
[DEBUG] 12:37:48.704 properties.go:GetInt:40 → max_depth: 2
[DEBUG] 12:37:48.705 properties.go:GetString:28 → 󰉓
[DEBUG] 12:37:48.705 properties.go:GetBool:22 → hide_root_location: false
[TRACE] 12:37:48.705 shell.go:PathSeparator() - 41ns
[TRACE] 12:37:48.705 shell.go:PathSeparator() - 42ns
[DEBUG] 12:37:48.705 properties.go:GetStringArray:52 → cycle: []
[DEBUG] 12:37:48.705 properties.go:GetString:28 → NO DATA
[TRACE] 12:37:48.705 shell.go:PathSeparator() - 41ns
[DEBUG] 12:37:48.705 properties.go:GetString:28 → /
[DEBUG] 12:37:48.705 properties.go:GetBool:22 → cycle_folder_separator: false
[DEBUG] 12:37:48.705 properties.go:GetString:28 → %s
[DEBUG] 12:37:48.705 properties.go:GetString:28 → %s
[DEBUG] 12:37:48.705 shell.go:CommandPath:615 → /Users/lucascosti/.nvm/versions/node/v20.10.0/bin/node
[TRACE] 12:37:48.705 shell.go:CommandPath(node) - 136.833µs
[TRACE] 12:37:48.705 shell.go:Pwd() - 84ns
[TRACE] 12:37:48.705 shell.go:HasCommand(node) - 139.458µs
[DEBUG] 12:37:48.705 shell_unix.go:IsWsl:35 → false
[TRACE] 12:37:48.705 shell_unix.go:IsWsl() - 2.875µs
[TRACE] 12:37:48.705 shell.go:StackCount() - 83ns
[TRACE] 12:37:48.705 shell_unix.go:DirIsWritable(/Users/lucascosti/repos/repo) - 13.666µs
[TRACE] 12:37:48.705 shell.go:TemplateCache() - 4.375µs
[TRACE] 12:37:48.705 shell.go:TemplateCache() - 3.417µs
[DEBUG] 12:37:48.705 properties.go:GetBool:22 → always_enabled: false
[DEBUG] 12:37:48.724 shell.go:RunCommand:594 → v20.10.0
[TRACE] 12:37:48.724 shell.go:RunCommand(node --version) - 19.227292ms
[DEBUG] 12:37:48.724 properties.go:GetString:28 → https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V{{ .Major }}.md#{{ .Full }}
[DEBUG] 12:37:48.724 text.go:Render:70 → Rendering template: https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V{{ .Major }}.md#{{ .Full }}
[TRACE] 12:37:48.724 shell.go:TemplateCache() - 209ns
[TRACE] 12:37:48.724 shell.go:Pwd() - 167ns
[DEBUG] 12:37:48.724 shell.go:FileContent:538 ↓
    lts/iron

[TRACE] 12:37:48.724 shell.go:FileContent(.nvmrc) - 83.167µs
[TRACE] 12:37:48.724 shell.go:TemplateCache() - 83ns
[DEBUG] 12:37:48.754 shell.go:RunCommand:594 ↓
    # branch.oid 5e8b34ce277dd1e2f288e374f3e5295b2ea22e05
    # branch.head main
    # branch.upstream origin/main
    # branch.ab +0 -0
[TRACE] 12:37:48.754 shell.go:RunCommand(git -C /Users/lucascosti/repos/repo --no-optional-locks -c core.quotepath=false -c color.status=false status -unormal --branch --porcelain=2) - 50.946167ms
[DEBUG] 12:37:48.754 properties.go:GetString:28 → 
[DEBUG] 12:37:48.754 properties.go:GetInt:40 → branch_max_length: 40
[DEBUG] 12:37:48.754 shell.go:HasFolder:508 → false
[TRACE] 12:37:48.754 shell.go:HasFolder(/Users/lucascosti/repos/repo/.git/rebase-merge) - 14.208µs
[DEBUG] 12:37:48.754 shell.go:HasFolder:508 → false
[TRACE] 12:37:48.754 shell.go:HasFolder(/Users/lucascosti/repos/repo/.git/rebase-apply) - 2.625µs
[DEBUG] 12:37:48.754 properties.go:GetString:28 → 
[DEBUG] 12:37:48.754 shell.go:HasFilesInDir:479 → false
[TRACE] 12:37:48.754 shell.go:HasFilesInDir(MERGE_MSG) - 11.292µs
[DEBUG] 12:37:48.754 shell.go:HasFilesInDir:479 → false
[TRACE] 12:37:48.754 shell.go:HasFilesInDir(CHERRY_PICK_HEAD) - 2.916µs
[DEBUG] 12:37:48.754 shell.go:HasFilesInDir:479 → false
[TRACE] 12:37:48.754 shell.go:HasFilesInDir(REVERT_HEAD) - 2.792µs
[DEBUG] 12:37:48.754 shell.go:HasFilesInDir:479 → false
[TRACE] 12:37:48.754 shell.go:HasFilesInDir(sequencer/todo) - 5.583µs
[DEBUG] 12:37:48.754 properties.go:GetString:28 → ≡
[DEBUG] 12:37:48.754 properties.go:GetBool:22 → fetch_upstream_icon: false
[TRACE] 12:37:48.754 shell.go:TemplateCache() - 250ns
[DEBUG] 12:37:48.754 text.go:Render:70 → Rendering template:  {{ .HEAD }} {{ .BranchStatus }}{{ if or .Working.Changed .Staging.Changed .StashCount }} {{ end }}{{ if gt .Working.Unmerged 0 }}<red>{{ .Working.Unmerged }}</>{{ end }}{{ if gt .StashCount 0 }}<#275EAA>󰉀{{ .StashCount }}</>{{ end }}{{ if or .Staging.Added .Staging.Modified .Staging.Deleted }}<#2431F5>{{ add (.Staging.Added) (.Staging.Modified) (.Staging.Deleted) }}</>{{ end }}{{ if gt .Working.Modified 0 }}<#A12916>{{ .Working.Modified }}</>{{ end }}{{ if gt .Working.Deleted 0 }}<#8a4801>{{ .Working.Deleted }}</>{{ end }}{{ if gt .Working.Untracked 0 }}<#555753>{{ .Working.Untracked }}</>{{ end }}
[DEBUG] 12:37:48.754 text.go:Render:70 → Rendering template: {{ if gt .Code 0 }}<red></>{{end}} 󱓞
[DEBUG] 12:37:48.754 text.go:Render:70 → Rendering template:  {{ if .Mismatch }}{{ .Full }} 󰦎 {{ .Expected }} {{ end }}
[DEBUG] 12:37:48.755 text.go:Render:70 → Rendering template:   {{ .Path }}
[TRACE] 12:37:48.755 shell.go:TemplateCache() - 208ns
[TRACE] 12:37:48.755 shell.go:Shell() - 83ns
[TRACE] 12:37:48.755 shell.go:TemplateCache() - 167ns
[TRACE] 12:37:48.755 shell.go:Shell() - 41ns
[TRACE] 12:37:48.755 shell.go:TemplateCache() - 83ns
[TRACE] 12:37:48.755 shell.go:Shell() - 83ns
[TRACE] 12:37:48.755 shell.go:TemplateCache() - 42ns
[ERROR] 12:37:48.755 shell.go:FileContent:534 → open /Users/lucascosti/repos/repo/.git/logs/refs/stash: no such file or directory
[TRACE] 12:37:48.755 shell.go:FileContent(/Users/lucascosti/repos/repo/.git/logs/refs/stash) - 11.625µs
[ERROR] 12:37:48.755 shell.go:FileContent:534 → open /Users/lucascosti/repos/repo/.git/logs/refs/stash: no such file or directory
[TRACE] 12:37:48.755 shell.go:FileContent(/Users/lucascosti/repos/repo/.git/logs/refs/stash) - 3.292µs
[TRACE] 12:37:48.755 shell.go:Shell() - 42ns
[DEBUG] 12:37:48.755 shell.go:Getenv:399 → iTerm.app
[TRACE] 12:37:48.755 shell.go:Getenv(TERM_PROGRAM) - 2.708µs
[DEBUG] 12:37:48.755 text.go:Render:70 → Rendering template: plain
[DEBUG] 12:37:48.755 text.go:Render:70 → Rendering template: powerline
[DEBUG] 12:37:48.755 text.go:Render:70 → Rendering template: powerline
[DEBUG] 12:37:48.755 text.go:Render:70 → Rendering template: powerline
[DEBUG] 12:37:48.755 text.go:Render:70 → Rendering template: {{ if or (.Working.Changed) (.Staging.Changed) }}#c4a000{{ end }}
[TRACE] 12:37:48.756 shell.go:TemplateCache() - 41ns
[DEBUG] 12:37:48.756 text.go:Render:70 → Rendering template: {{ if or (.Working.Changed) (.Staging.Changed) }}#323734{{ end }}
[TRACE] 12:37:48.756 shell.go:TemplateCache() - 84ns
[DEBUG] 12:37:48.756 text.go:Render:70 → Rendering template: powerline
[TRACE] 12:37:48.756 shell.go:Shell() - 42ns
[TRACE] 12:37:48.756 shell.go:Shell() - 0s
[TRACE] 12:37:48.756 shell.go:Shell() - 42ns
[TRACE] 12:37:48.756 shell.go:Shell() - 42ns
[TRACE] 12:37:48.756 shell.go:Flags() - 42ns
[TRACE] 12:37:48.756 shell.go:Pwd() - 125ns
[TRACE] 12:37:48.756 shell.go:GOOS() - 41ns
[DEBUG] 12:37:48.756 segment.go:SetEnabled:512 → Segment: Text
[TRACE] 12:37:48.756 shell.go:TemplateCache() - 42ns
[DEBUG] 12:37:48.756 text.go:Render:70 → Rendering template: ❯
[TRACE] 12:37:48.756 shell.go:Shell() - 42ns
[DEBUG] 12:37:48.756 shell.go:Getenv:399 → iTerm.app
[TRACE] 12:37:48.756 shell.go:Getenv(TERM_PROGRAM) - 1.625µs
[DEBUG] 12:37:48.756 text.go:Render:70 → Rendering template: plain
[TRACE] 12:37:48.756 shell.go:Shell() - 41ns
[TRACE] 12:37:48.756 shell.go:Shell() - 42ns
[TRACE] 12:37:48.756 shell.go:Shell() - 41ns
[TRACE] 12:37:48.756 shell.go:Shell() - 42ns
[TRACE] 12:37:48.756 shell.go:Flags() - 41ns
[DEBUG] 12:37:48.756 shell.go:Getenv:399 → NO DATA
[TRACE] 12:37:48.756 shell.go:Getenv(OMP_CACHE_DIR) - 1.917µs
[DEBUG] 12:37:48.756 shell.go:Getenv:399 → NO DATA
[TRACE] 12:37:48.756 shell.go:Getenv(XDG_CACHE_HOME) - 1.208µs
[TRACE] 12:37:48.756 shell_unix.go:CachePath() - 14.833µs
[TRACE] 12:37:48.756 shell.go:Flags() - 42ns
```

--- a/src/segments/node.go
+++ b/src/segments/node.go
@@ -69,6 +69,33 @@ func (n *Node) matchesVersionFile() (string, bool) {
 		return "", true
 	}
 
+	fileVersion = strings.TrimSpace(fileVersion)
+
+	if strings.HasPrefix(fileVersion, "lts/") {
+		fileVersion = strings.ToLower(fileVersion)
+		codeName := strings.TrimPrefix(fileVersion, "lts/")
+		switch codeName {
+		case "argon":
+			fileVersion = "4.9.1"
+		case "boron":
+			fileVersion = "6.17.1"
+		case "carbon":
+			fileVersion = "8.17.0"
+		case "dubnium":
+			fileVersion = "10.24.1"
+		case "erbium":
+			fileVersion = "12.22.12"
+		case "fermium":
+			fileVersion = "14.21.3"
+		case "gallium":
+			fileVersion = "16.20.1"
+		case "hydrogen":
+			fileVersion = "18.19.0"
+		case "iron":
+			fileVersion = "20.10.0"
+		}
+	}
+
 	re := fmt.Sprintf(
 		`(?im)^v?%s(\.?%s)?(\.?%s)?$`,
 		n.language.version.Major,

--- a/src/segments/node_test.go
+++ b/src/segments/node_test.go
@@ -11,10 +11,10 @@ import (
 
 func TestNodeMatchesVersionFile(t *testing.T) {
 	nodeVersion := version{
-		Full:  "1.2.3",
-		Major: "1",
-		Minor: "2",
-		Patch: "3",
+		Full:  "20.10.0",
+		Major: "20",
+		Minor: "10",
+		Patch: "0",
 	}
 	cases := []struct {
 		Case            string
@@ -23,14 +23,17 @@ func TestNodeMatchesVersionFile(t *testing.T) {
 		RCVersion       string
 	}{
 		{Case: "no file context", Expected: true, RCVersion: ""},
-		{Case: "version match", Expected: true, ExpectedVersion: "1.2.3", RCVersion: "1.2.3"},
-		{Case: "version match with newline", Expected: true, ExpectedVersion: "1.2.3", RCVersion: "1.2.3\n"},
+		{Case: "version match", Expected: true, ExpectedVersion: "20.10.0", RCVersion: "20.10.0"},
+		{Case: "version match with newline", Expected: true, ExpectedVersion: "20.10.0", RCVersion: "20.10.0\n"},
 		{Case: "version mismatch", Expected: false, ExpectedVersion: "3.2.1", RCVersion: "3.2.1"},
-		{Case: "version match in other format", Expected: true, ExpectedVersion: "1.2.3", RCVersion: "v1.2.3"},
-		{Case: "version match without patch", Expected: true, ExpectedVersion: "1.2", RCVersion: "1.2"},
-		{Case: "version match without patch in other format", Expected: true, ExpectedVersion: "1.2", RCVersion: "v1.2"},
-		{Case: "version match without minor", Expected: true, ExpectedVersion: "1", RCVersion: "1"},
-		{Case: "version match without minor in other format", Expected: true, ExpectedVersion: "1", RCVersion: "v1"},
+		{Case: "version match in other format", Expected: true, ExpectedVersion: "20.10.0", RCVersion: "v20.10.0"},
+		{Case: "version match without patch", Expected: true, ExpectedVersion: "20.10", RCVersion: "20.10"},
+		{Case: "version match without patch in other format", Expected: true, ExpectedVersion: "20.10", RCVersion: "v20.10"},
+		{Case: "version match without minor", Expected: true, ExpectedVersion: "20", RCVersion: "20"},
+		{Case: "version match without minor in other format", Expected: true, ExpectedVersion: "20", RCVersion: "v20"},
+		{Case: "lts match", Expected: true, ExpectedVersion: "20.10.0", RCVersion: "lts/iron"},
+		{Case: "lts match upper case", Expected: true, ExpectedVersion: "20.10.0", RCVersion: "lts/Iron"},
+		{Case: "lts mismatch", Expected: false, ExpectedVersion: "8.17.0", RCVersion: "lts/carbon"},
 	}
 
 	for _, tc := range cases {

